src/common/modules/Logger.js

```diff
@@ -13,6 +15,22 @@ const MESSAGE_LEVEL_NAME = Object.freeze({
     [MESSAGE_LEVEL.SUCCESS]: "SUCCESS"
 });
 
+/**
+ * Copy nested objects to ensure a properly output.
+ *
+ * @function
+ * @param  {array} args
+ * @returns {Object}
+ */
+function prepareObjectsForLogging(args) {
+    for(const [index, value] of args.entries()) {
```

Comment:
Oh, awesome, you use destructuring here. :smile: 

But this indicates `args` is an object and no array? This can be true and I just completely miss [how the spread syntax worked](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Spread_syntax). Or, in this case, rather [rest](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters).

Reply:
I searched for ways to iterate over an array and also having a index or some other way to exchange the values in it. To me this solution looked the cleanest. But the same could be achieved by using a ```for``` loop.

You are right ```args.entries()``` kind of indicates that ```args``` is an object. So perhaps this solution is still acceptable because ```args``` is kind of an object ? ([The arguments object is an Array-like object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments))

Reply:
No, it's okay, I guess. [entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/entries) is a property of an array. I also looked and found only anther way using `.forEach()`, so if you wanted, you may use this, but personally I don't care. This solution here works, too.

BTW `args` is not `arguments` anymore, as we use the rest operator (as linked above), which converts it into an array.